src/shared/components/post/create-post.tsx

```diff
@@ -293,6 +296,24 @@ export class CreatePost extends Component<
               isNsfwCommunity={selectedCommunityIsNsfw}
             />
           </div>
+          <div className="d-block d-md-none">
+            <button
+              className="btn btn-secondary d-inline-block mb-2 me-3"
+              onClick={this.handleShowSidebarMobile}
+            >
+              {I18NextService.i18n.t("sidebar")}{" "}
+              <Icon
+                icon={
+                  this.state.showSidebarMobile ? `minus-square` : `plus-square`
+                }
+                classes="icon-inline"
+              />
+            </button>
+            {this.state.showSidebarMobile && this.sidebar()}
+          </div>
```

Comment:
This shows the sidebar button below all the input fields which is easy to miss. Another option would be to put sidebar button next to the community selector inside PostForm but thats a bit more complicated to implement.

Reply:
What you have is good, and I can't think of any cleaner way to do it for mobile. 

On the Home and Community pages, we put all these buttons at the top, but it seems weird to do so before a community is picked.